tests/scaffoldings/repo_merge/docker-compose.yaml

```diff
@@ -6,6 +6,7 @@ services:
       args:
         COMPILE: "false"
         ODOO_VERSION: $ODOO_MINOR
+        PIP_INSTALL_ODOO: "false" # ensure build.d/700-odoo-install does not fail
```

Comment:
What is this?

Reply:
to me it looks like the `pip install odoo` is disabled for this because /tmp/fake-odoo is not installable as an odoo package

Reply:
Yes, it's like @ap-wtioit  said. Perhaps I should have made the comment more explicit.